modules/Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java

```diff
@@ -720,6 +652,90 @@ else if (diffz < 0) {
         }
     }
 
+    //Returns a given craft's status as a pair of booleans.
+    //Left is sinking, right is disabled.
+    public Pair<Boolean, Boolean> checkCraftStatus(Craft craft) {

```

Comment:
I'm not sure if there is a better way to do this, maybe cc can give us a smart idea.

Reply:
perhaps a craftStatus class with two variables, isSinking and isDisabled?